src/asciidoctor-revealjs.tmpl.js

```diff
@@ -0,0 +1,46 @@
+(function (Opal) {
+  function initialize (Opal) {
+//OPAL-GENERATED-CODE//
+  }
+
+  var mainModule
+
+  function resolveModule () {
+    if (!mainModule) {
+      checkAsciidoctor()
+      initialize(Opal)
+      mainModule = Opal.const_get_qualified(Opal.Asciidoctor, 'Revealjs')
+    }
+    return mainModule
+  }
+
+  function checkAsciidoctor () {
+    if (typeof Opal.Asciidoctor === 'undefined') {
+      throw new TypeError('Asciidoctor.js is not loaded')
+    }
+  }
+
+  /**
+   * @return {string} Version of this extension.
+   */
+  function getVersion () {
+    return resolveModule().$$const.VERSION.toString()
+  }
+
+  /**
+   * Registers the reveal.js converter.
+   */
+  function register () {
+    return resolveModule()
```

Comment:
The function resolves the module (ie. initialize the module only once).

Reply:
I'm not the one you should ask that unfortunately :(

Reply:
@obilodeau I think it's fine because it can be useful 😉